src/templates/post.jsx

```diff
@@ -26,7 +27,7 @@ export default class PostTemplate extends React.Component {
               <h1>{post.title}</h1>
               <h6>
                 {`Posted by ${post.author} on `}
-                <span>{post.ddate}</span>
+                <span>{moment(date).format("MMMM DD, YYYY")}</span>
```

Comment:
This is a bit of duplicated code using the same format string in different places. I don't know where we should derive this display date (do it earlier and pass it on, or do it as late as possible?)

I'd like to look at this again when we rework components and queries and reduce overall code duplication.